### PR TITLE
SPOI-8548: Example to use jdbc poll input operator

### PR DESCRIPTION
**Pull request by DT-Priyanka**

### CODE REVIEW

**Comment by DT-Priyanka** [comment]
@devtagare @bhupeshchawda  please review and merge.

**Comment by devtagare** [comment]
Please add a test for JdbcPollerApplication under examples/tutorials/jdbcIngest/src/test/java/com/example/mydtapp/ to validate the application results. 
For each of the examples its expected that the user can download, build and run them using the test case provided without having to make any changes to the configs.

File: tutorials/jdbcIngest/src/main/resources/META-INF/pollerProperties.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0"?>
```
Comment (amberarrow) [inline tutorials/jdbcIngest/src/main/resources/META-INF/pollerProperties.xml]: Is the name of this XML file correct ? From the documentation at
http://docs.datatorrent.com/application_packages/
it seems that application-specific properties files should be named properties-{appName}.xml

Please run the example on a cluster and double check that the README.md has complete and accurate instructions on what the user needs to do to successfully run the application on a cluster -- we want the user experience to be as smooth and painless as possible when running these examples both in local mode and on a cluster.

**Comment by DT-Priyanka** [comment]
@devtagare  added test as well as updated readme and properties file name as per suggestions by @amberarrow

**Comment by DT-Priyanka** [comment]
@devtagare  can you please review this?

**Comment by DT-Priyanka** [comment]
@yogidevendra  can you please review and merge?

**Comment by devtagare** [comment]
@DT-Priyanka - looks good to merge.Can you bump up the Malhar version to 3.5.0 (since its released now)

File: tutorials/jdbcIngest/src/test/java/com/example/mydtapp/JdbcPollerApplicationTest.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+      conf.set("dt.application.PollJdbcToHDFSApp.operator.JdbcPoller.port.outputPort.attr.TUPLE_CLASS",
+          "com.example.mydtapp.PojoEvent");
+      conf.set("dt.application.PollJdbcToHDFSApp.operator.Writer.filePath", OUTPUT_DIR_NAME);
+
+      lma.prepareDAG(new JdbcPollerApplication(), conf);
+      LocalMode.Controller lc = lma.getController();
+      lc.runAsync();
+
+      // wait for output files to roll      
+      Thread.sleep(5000);
```
Comment (tweise) [inline tutorials/jdbcIngest/src/test/java/com/example/mydtapp/JdbcPollerApplicationTest.java]: @DT-Priyanka this is the same problem that was discussed elsewhere. How do we know that after 5000 seconds that application was done?

Comment (DT-Priyanka) [inline tutorials/jdbcIngest/src/test/java/com/example/mydtapp/JdbcPollerApplicationTest.java]: @tweise in this case we are inserting very few records i.e. 10 for testing. But surely in rare case it won't be done in given time. What's the good solution to address this problem? Should I keep waiting on some trigger before stopping application? I have done in in other app (example for file reader).

Comment (tweise) [inline tutorials/jdbcIngest/src/test/java/com/example/mydtapp/JdbcPollerApplicationTest.java]: Please see discussion on dev@ list regarding this topic. Please see other tests for examples of the recommended solution. Tests should exit as soon as expected results are in and assert the same.

Comment (DT-Priyanka) [inline tutorials/jdbcIngest/src/test/java/com/example/mydtapp/JdbcPollerApplicationTest.java]: okay, I would address it soon. Creating a jira for reference: https://datatorrent.atlassian.net/browse/SPOI-9191

**Pull request merged**